### ISSUE TITLE
Questions, Comments, Feedback

### ISSUE THREAD

**Issue by LarryJoy** [inventree/InvenTree#846]
Hello Oliver,
Now that I am able to get into InvenTree and start using the program, here come the the questions, comments, and feedback.
• I see that you use the term "BOM", meaning Bill-of-Material, as many people do, and I wonder what your reference is? ANSI/ASME Y14.34 **Associated Lists** has:
![image](https://user-images.githubusercontent.com/18520951/83350037-23dce880-a307-11ea-8df5-eb4e47d82a4f.png)
And this note:
![image](https://user-images.githubusercontent.com/18520951/83350065-643c6680-a307-11ea-91f5-ea997ae731f9.png)
I prefer and use the term "parts list" (PL) myself but can get along with the term "BOM".
• In the BOM "Reference" column, can this be sorted to be in alpha-numeric order?
• There are two varieties of parts lists (PLs)/BOMs.
1) Is the engineering or design type that has one line for each instance of a part. This comes in handy when trouble shooting an assembly as it is easy to find the component in ref des alpha numeric order. For design purposes you can easily see the last ref des used if you need to add another component and you can see which ref des is not used.
2) Is the manufacturing/purchasing (procurement) style where each line could have multiple instances of a component and thus a total number of that component that is needed. However, it is difficult to pick out a particular ref des when there are many items.
When handling PLs/BOMs a program should be able to readily go back and forth between these two styles, especially these days with modern computer technology.

• You can view my series of articles about reference designations at <https://techexplorations.com/reference-designations-for-electrical-and-electronics-parts-and-equipment/>. This website is out of Sydney, Australia.
--Larry

**Comment by LarryJoy** [comment]
Oliver, I see by doodling around that the BOM function does have a sorting function for the Reference column and will do a sort on part numbers, except it doesn't sort correctly for my part numbers. In the Parts section the sort on part numbers does not sort correctly either.
--Larry

**Comment by eeintech** [comment]
Hey @LarryJoy, thanks for your feedback. The term BoM is very commonly used and I'm able to sort parts in a BoM by any field shown in the table.

If you have any other comment/feedback, do you mind closing this issue and opening a new one with more focus on the actual problem or improvement you are proposing? Thanks 😃

**Comment by LarryJoy** [comment]
Yes, I too am able to sort correctly on any column/field in a BoM, just have to click on the heading more than once.
--Larry
